app/controllers/templates_controller.rb

```diff
@@ -33,7 +33,7 @@ def update
   end
 
   def show_form
-    @template = Template.find params[:id]
+    @template = Template.find params[:id] || Template.where(:active => true).first
```

Comment:
This isn't actually fail handling - this is here because in this PR, templates are associated with campaign pages. templates/show_form gets requested either when creating a new page (in which case we want to load the first active template as the first option on the page), or when editing the page (which is when template is passed as a template id in the parameters). I used this to load a default template vs. a specified template when navigating to campaign_pages/new (which otherwise shows up with no template loaded) vs campaign_pages/:id/edit, but I handled that in the campaign pages controller instead, so this line is redundant.